packages/primereact/package.json

```
@@ -0,0 +1,97 @@
+{
+  "name": "@rjsf/primereact",
+  "version": "6.0.0-alpha.0",
+  "description": "PrimeReact theme, fields and widgets for react-jsonschema-form",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "build:ts": "tsc -b",
```

To improve ESM support you need to change this to:
```suggestion
    "build:ts": "tsc -b tsconfig.build.json && tsc-alias -p tsconfig.build.json",
```

And add a `tsconfig.build.json` that looks like:
```
{
  "extends": "../../tsconfig.build.json",
  "compilerOptions": {
    "outDir": "./lib"
  },
  "files": [],
  "references": [
    {
      "path": "./src"
    }
  ],
  "tsc-alias": {
    "resolveFullPaths": true,
    "verbose": true,
  }
}
```